lib/settings/settings.js

```diff
@@ -103,7 +103,11 @@ class Settings {
   get testWorkersEnabled() {
     const {test_workers} = this.settings;
 
-    return this.argv.parallel === true || test_workers === true || isObject(test_workers) && test_workers.enabled;
+    if (this.argv.parallel === false ||  test_workers.enabled === false) {
```

Comment:
If there is any chance `test_workers`'s value will be null or undefined, it will break.

Reply:
fixed